src/main.cpp

```diff
@@ -2816,7 +2827,10 @@ void update(void)
         process_action_points();
         player = get_my_player();
         if (player->view_mode == PVM_CreatureView)
-            update_flames_nearest_camera(player->acamera);
+        {
+            struct Thing *thing = thing_get(player->controlled_thing_idx);
+            update_flames_nearest_thing(thing);
```

Comment:
I think it would be neater to just do it like this:
`update_flames_nearest_thing(player->controlled_thing_idx);`, and find the Thing inside the function itself.